source/backtraceMetric.ts

```diff
@@ -0,0 +1,226 @@
+import { BacktraceClientOptions } from '.';
+import { uuid } from './utils';
+
+declare const __VERSION__: string;
+
+/**
+ * Function to create instance of BacktraceMetric and ping.
+ */
+export default function backtraceMetric(
+  configuration: BacktraceClientOptions,
+): void {
+  new BacktraceMetric(configuration).ping();
+}
+
+/**
+ * Handles Backtrace Metrics.
+ * TODO: Consider converting to a static class since an instance is only created to call a single function once.
+ */
+export class BacktraceMetric {
+  public universe;
+  public token;
+  public timeout = 15000; // fifteen seconds in miliseconds
+
+  /** Seconds since epoch. */
+  public readonly CURRENT_TIMESTAMP: number = Math.floor(
+    new Date().getTime() / 1000,
+  );
+  /** Thirty minutes in seconds. */
+  public readonly THIRTY_MINUTES = 1800;
+
+  public readonly userAgent = navigator.userAgent;
```

Comment:
Those informations are already available in the backtraceData/backtraceReport object. Wouldn't it be better to just store it in one place?

Reply:
Agreed, reverted that line

Reply:
Yes I think so too. I may write them to the `utility` file and fetch them there. Still WIP